lib/puppet/provider/package/chocolatey.rb

```diff
@@ -52,6 +55,10 @@ def self.chocolatey_command
     chocopath
   end
 
+  def self.chocolatey_relocation_in_progress?
+     ENV['ChocolateyInstall'].downcase == 'c:\chocolatey' && file_exists?('C:\ProgramData\chocolatey\bin\choco.exe')
+  end
```

Comment:
Actually there is a much better way to get the value that is always accurate. See the choco_install_path fact.